src/java.base/share/classes/sun/net/www/http/HttpClient.java

```diff
@@ -165,13 +180,26 @@ int getKeepAliveTimeout() {
         return keepAliveTimeout;
     }
 
+    static String normalizeCBT(String s) {
+        if (s == null || ! (s.equals("always") ||
+                s.equals("never") || s.startsWith("domain:"))) {
+            return "never";
+        } else {
+            logError("Unexpected value for \"jdk.https.negotiate.cbt\" system property");
+            return s;
```

Comment:
It was being handled elsewhere as "never". But, I agree it would be clearer to normalise it to "never" here.

Reply:
Sorry, I should have checked back to the source rather than the snippet quoted. The problem is that the logError call is in the wrong place. It should be before line 186.

Reply:
Ah! Yes - I was also bitten again by the negation in the `if` too.
The presence of `logError` in the body of the `if` will make it clearer :-)